descope/common.py

```diff
@@ -21,7 +21,7 @@
 
 
 class EndpointsV1:
-    publicKeyPath = "/v1/keys"
+    publicKeyPath = "/v2/keys"
```

Comment:
Should be under `EndpointsV2`

![image](https://user-images.githubusercontent.com/5429033/206675586-0f43d5d6-9985-4227-a77b-837f0b42889f.png)

Reply:
Fixed